db/migrate/20220105085729_split_customers_name.rb

```diff
@@ -1,9 +1,9 @@
-require 'concerns/address_display'
-require 'spree/address'
-
 class SplitCustomersName < ActiveRecord::Migration[6.1]
+  class Spree::DummyAddress < ApplicationRecord
+    self.table_name = 'spree_addresses'
```

Comment:
ActiveRecord generates the table name from the class name without module scope. So this would work, too:
```suggestion
  class SpreeAddress < ApplicationRecord
```

And then you need `class_name: "SplitCustomersName::SpreeAddress"` below.

Reply:
If I do this, I am getting this error:

Reply:
Nice trick :)